azurelinuxagent/common/cgroupconfigurator.py

```diff
@@ -417,21 +448,36 @@ def supported(self):
             return self._cgroups_supported
 
         def enabled(self):
-            return self._cgroups_enabled
+            return self._agent_cgroups_enabled
```

Comment:
just adding a reminder wrt the conversation we had about global vs agent&extensions flags

Reply:
addressed

Reply:
addressed